packages/apps-config/src/ui/logos/index.ts

```diff
@@ -421,6 +423,7 @@ export const chainLogos = Object.entries({
 export const nodeLogos = Object.entries({
   'Acala Node': chainAcala,
   'Ajuna Node': nodeAjuna,
+  'Amplitude': chainAmplitude,
```

Comment:
```suggestion
  Amplitude: chainAmplitude,
```

This should make [CI happy](https://github.com/polkadot-js/apps/runs/7132831326?check_suite_focus=true#step:3:702). (Alternatively run `yarn lint` locally for auto-fixes)